### PR TITLE
JSON loading and displaying

### PR DESCRIPTION
**Pull request by aoswalt**
Non-default master branch because of early mistake and cannot fix because of not owning the repo.

### CODE REVIEW

File: food.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+'use strict';
```
Comment (sscotth) [inline food.js]: don't mix single quotes a double quotes. Also, place your `"use strict";` statement inside the IIFE.

``` js
var Food = (function(food) {
  "use strict";

  var targetElement = document.getElementById("food-table-body");
```

File: index.html [hunk 2]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>SPA - Pet Food</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <table class="food-table" rules="groups">
```
Comment (sscotth) [inline index.html]: where did you find rules? Its been deprecated 👎 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/table#attr-rules

File: food.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+
+  function insertFood(target, foodData) {
+    var brandList = foodData[Object.keys(foodData)[0]];
+    var html = "";
+
+    brandList.forEach(function(brand) {
+      var brandName = brand.name.replace(/\b(\w)/g, function(x) { return x.toUpperCase(); });
+      html += `<tr><td>${brandName}</td>`;
+
+      html += `<td>${brand.hasOwnProperty("breeds") ? brand.breeds.join("<br>") : ""}</td>`;
```
Comment (sscotth) [inline food.js]: Can you do this without `<br>` tags?

**Comment by aoswalt** [comment]
Old

**Pull request closed without merging**